cypress/e2e/tests/migration/applicationinventory/assessment/archetype_association.test.ts

```diff
@@ -63,9 +70,21 @@ describe(["@tier2"], "Tests related to application-archetype association ", () =
         archetype.create();
         cy.wait(2 * SEC);
 
-        //Automates Polarion MTA-499 Verify multiple applications inherit assessment and review inheritance from an archetype
+        /*Automates Polarion MTA-499 Verify multiple applications inherit assessment and review inheritance from an archetype
+          and Polarion MTA-2464 Assess archetype with multiple questionnaires */
         archetype.perform_review("low");
         archetype.perform_assessment("low", stakeholders);
+        archetype.validateAssessmentField("Unknown");
```

Comment:
After "low" assessment , shouldn't the field be validated for "Low" instead of "unknown" ?

Reply:
Since the Archetype is being assessed by 2 required questionnaires, the 'Assessment' field shows 'Unknown' until all required questionnaires have been taken. Once all the required questionnaires have been taken, the 'Assessment' field reflects the correct risk assessment .